integration_tests/src/main/python/hash_aggregate_test.py

```diff
@@ -1844,11 +1844,9 @@ def test_std_variance_partial_replace_fallback(data_gen,
     null_gen] + array_gens_sample + struct_gens_sample
 @ignore_order(local=True)
 @pytest.mark.parametrize('data_gen',  gens_for_max_min, ids=idfn)
-def test_min_max_for_single_level_struct(data_gen):
+def test_min_max_for_nested_types(data_gen):
     df_gen = [
-        ('a', StructGen([
-                ('aa', data_gen),
-                ('ab', data_gen)])),
+        ('a', data_gen),
         ('b', RepeatSeqGen(IntegerGen(), length=20))]
 
```

Comment:
You added in support for a reduction on min/max (no group by keys).  Could we also add in a test for that case too?

Reply:
Yes this test function already has both reduction and groupby tests.